internal/future/app/mmlogic/mmlogic_service.go

```diff
@@ -15,31 +15,139 @@
 package mmlogic
 
 import (
+	"time"
+
+	"encoding/json"
+
+	"github.com/gomodule/redigo/redis"
+	"github.com/sirupsen/logrus"
+	"open-match.dev/open-match/internal/config"
 	"open-match.dev/open-match/internal/future/pb"
+	"open-match.dev/open-match/internal/future/storage"
+	"open-match.dev/open-match/internal/set"
+
+	"fmt"
+)
+
+var (
+	logger = logrus.WithFields(logrus.Fields{
```

Comment:
There's a convention to use the filename as part of the logger name. That way each file has it's own logger. This might change in the future.